src/app/components/media/ReportDesigner/ReportDesignerComponent.js

```diff
@@ -227,7 +227,7 @@ const ReportDesignerComponent = (props) => {
         onStateChange={(action, state) => { handleSave(action, state); }}
         prefixUrl={prefixUrl}
       />
-      <Box display="flex" width="1">
+      <Box display="flex" width="1" style={{ borderLeft: 'solid 2px var(--grayBorderMain)' }}>
```

Comment:
This was added temporarily, to keep the item page looking the same until we get around to those layout changes